python-sdk/src/astro/sql/operators/run_query.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Callable, Optional
+
+from airflow.decorators.base import get_unique_task_id
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from astro.databases import create_database
+
+
+class RunQueryOperator(BaseOperator):
```

Comment:
@tatiana @dimberman  What do you think about the naming?

Reply:
putting the previous discussion about this naming here for reference https://github.com/astronomer/astro-sdk/pull/673#discussion_r949811361

Reply:
@pankajastro @kaxil how does this operator relate to the RunRawSQL? It feels both are very similar. Do we want to keep both of them? Do we want to deprecate one of them?

Reply:
@tatiana Mainly ff we don't pass the handler in `RawSQLOperator` then it returns `None` but RunQueryOperator returns fetchall response.

Reply:
@kaxil any thoughts on the above i.e using `RawSQLOperator` after some modifications?

Reply:
@pankajastro and I agreed that we'll be using the `RawSQLOperator` to implement `get_value_list`, defining the handler